datastore/mongodb/filter_dimensions.go

```diff
@@ -342,7 +342,7 @@ func (c *Client) DeleteFilterDimension(ctx context.Context, filterID, dimensionN
 
 	// check the available filter dimensions.
 	// 1. A filter should always have a minimum of 2 dimensions
-	if len(filter.Dimensions) < 3 {
+	if len(filter.Dimensions) < 1 {
```

Comment:
Should this have not gone down to 2 rather than 1? Also comment will need updating

Reply:
done